README.md

```diff
@@ -313,6 +313,30 @@ formState.values.doesNotExist
 
 By default, `useFormState` will use the type `any` for the form state and its inputs if no type argument is provided. Therefore, it is recommended that you provide one.
 
+By default, the `errors` property will contain strings.  If you return complex error objects from custom validation, you can provide an error type:
+
+```ts
+interface I18nError {
+  en: string;
+  fr: string;
+}
+
+const [formState, { text }] = useFormState<LoginFormFields, StateErrors<LoginFormFields, I18nError>();
```

Comment:
I'm not sure I'd make `StateErrors` part of the library as I don't see a huge value in including it – how about we get rid of it in favor of the simpler, more explicit approach you included below?

```ts
interface I18nError {
  en: string;
  fr: string;
}

interface LoginFormErrors {
  username?: string | I18nError; // StateErrors can be replicated like this
  password?: string;
}

useFormState<LoginFormFields, LoginFormErrors>
```

This reduces the complexity and keeps the codebase a tiny bit easier to manage.

Reply:
Yeah, sounds like a good call.  :)  I'll make that change.